cfg_model: 1classesV3.cfg
weights_model: 1classesV3_3000.weights
object_threshold: 0.1
class_names: ['turtle_bot']
class_widths: [40.64] # unused for comp1
class_heights: [40.64] # unused for comp1